@here/harp-mapview/lib/text/Placement.ts

```diff
@@ -429,6 +429,11 @@ function placePointLabelChoosingAnchor(
     // Store label state - persistent or new label.
     const persistent = labelState.visible;
 
+    // Text elements with multi-anchor placement require their own layout style.
+    // Provide private copy of label layout and attach it to current label and text canvas.
+    label.layoutStyle = label.layoutStyle!.clone();
```

Comment:
so far let's do it like this, but after this PR we'll need to find a way to avoid cloning the style here, it can be a big performance impact.